### DISCUSSION TITLE
How to rectify the error: segmentation fault

### DISCUSSION THREAD

**Discussion by kumaranvenk** [neperfepx/neper#847] [Q&A] [answered]
I am facing error while writing the code:

neper -T -n 15 -dim 2 -morpho voronoi -domain 'square(200,200)' -morphooptiini 'coo:file(c15w),weight:file(c15c)'

**c15w:**

0.6507 16.8628
2.0379 12.6145
4.0162 10.0003
0.6216 2.6024
0.889 0.6312
10.3675 6.6833
2.1003 16.5929
19.1953 15.4247
17.6058 6.5424
19.2485 9.0578
21.1099 12.0493
22.8446 15.5153
23.0493 10.3715
20.3289 7.5873
21.8995 1.2219

**c15c**

13.47
3.77
5.19
2.06
1.48
6.79
2.78
20.04
10.79
14.35
22.07
16.55
7.38
8.24
3.64

**Comment by rquey** [comment, marked as answer]
This discussion has been closed as obsolete. Please repost if necessary.
